src/OTAHandlers.cpp

```diff
@@ -92,14 +95,6 @@ void OTAHandlers::initFlash()
         DBG_vPrintf(TRUE, "OTAHandlers::initFlash(): Failed to allocate endpoint OTA space (can be ignored for non-OTA builds). status=%d\n", status);
 }
 
-void OTAHandlers::vDumpOverridenMacAddress()
```

Comment:
I would keep this method, just in case. Please move it to DumpFunctions.cpp for consistency